test/unit/storage/test_json_requests.rb

```diff
@@ -49,4 +49,12 @@ def test_unescaped_slashes_in_url
     assert_match(/a\/b\/c/, url,
                  "slashes should not be escaped with '%2F'")
   end
+
+  def test_unescaped_pluses_in_url
+    url = @client.get_object_https_url("bucket",
+                                      "a+c.ext",
+                                      Time.now + 2 * 60)
+    assert_match(/a\+c/, url,
+                 "pluses should not be escaped with '%2B'")
+  end
```

Comment:
Nice spec! And it's nice to see the existing specs about slashes not being escaped, and spaces being escaped, are continuing to work.